start-from-zero/sass-architecture/sass/components/_accordion.sass

```diff
@@ -0,0 +1,143 @@
+.reveal
+    display: block
+    background: blue
+    height: 60px
+    width: 60px
+
+.hidethis
+    display: none
+
+.details-list
+    list-style-type: none
+
+#accordion
+    padding: 0
+    margin: 0
+
+.details-textarea
+    display: inline-flex
+    width: 100%
+    border: 1px solid $main-color
+    margin-bottom: 25px
+    position: relative
+    
+    p
+        padding-left: 20px
+
+    &:before
```

Comment:
Minor. Pseudo-elements should start with `::` in CSS 3.0 format. Single-colon (`:`) format is 2.1 format will still work, but since it's antiquated, it's a good practice to replace it with the newer format.

Reply:
Ok, thanks for pointing this out. I fixed it.